docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -1223,12 +1223,13 @@ the timeout expires, the request fails and returns an error. Defaults to `30s`.
 Can also be set to `-1` to indicate that the request should never timeout.
 end::master-timeout[]
```

Comment:
It would be great to also explain how `master_timeout` is computed. By reading the code, it seems to start when the task is added to the queue and expires if the task does not get processed by the master. And I believe during this waiting, the task is visible via the [PendingTasks API](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-pending.htm)?